app/assets/stylesheets/pages/_home.scss

```diff
@@ -2,3 +2,57 @@
  * CSS code specific to home page
  * (margin/padding adjustments)
  * ------------------------------------- */
+ .banner {
+  background: center no-repeat;
+  background-size: cover !important;
+  position: relative;
+  color: white;
+  text-align: center;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: 'Source Sans Pro', sans-serif;
+}
+.banner h1 {
```

Comment:
You can use the css nested attributes examples: 
.banner{ 
some css code
           h1{
                some more css code
              }
}